src/main/resources/public/js/app.js

```diff
@@ -76,6 +76,10 @@ function scan(){
    request.done(function( currModel ) {
      displayGameState(currModel);
      gameModel = currModel;
+     if(gameModel.scanResult){
+     alert("Scan found at least one Ship")}
+     else{
+     alert("Scan found no Ships")}
```

Comment:
It's very good that you found this bug in our code. The scan message was being displayed every time because it was in the wrong place. This code would be better if we didn't need an alert window and could just display the message(of a ship found or not found) somewhere else on the screen. That way the player wouldn't have anything getting in the way of their game.